cmd/sqlflow/main.go

```diff
@@ -356,7 +367,12 @@ func main() {
 		if !*noAutoCompletion {
 			// TODO(lorylin): get autocomplete dicts for sqlflow_models from sqlflow_server
 		}
-		runPrompt(func(stmt string) { runStmt(*serverAddr, stmt, true, *ds) })
+		runPrompt(func(stmt string) {
```

Comment:
maybe we should move this logic to [there](https://github.com/sql-machine-learning/sqlflow/blob/2ad28c15dd7b773f19aa2991a8de907bde649ca4/cmd/sqlflow/main.go#L186) and keep the code clean here.

btw, we may say 'Goodby!' as we do in response to `ctrl+D` action

Reply:
Thanks for suggestions! I have once moved this logic to the position you mentioned. However, I keep `os.Exit(0)` to the position in this PR after thinking. My concern is that `runStmt` should be a function to execute SQL statements. So it should report execution results or errors after this function is invoked, instead of considering the exit cases. Therefore, I think that calling `os.Exit(0)` inside `runStmt` may not be a good idea.

Reply:
To say `Goodbye!` before `os.Exit(0)` is a good idea! I would add it asap in the next PR.

Reply:
There is another special case `use db`, like `exit`, we can process these client side command together as mentioned above.